.tekton/determine-image-tag-task.yaml

```diff
@@ -31,7 +31,7 @@ spec:
       name: workdir
   steps:
   - name: use-trusted-artifact
-    image: quay.io/redhat-appstudio/build-trusted-artifacts:latest@sha256:4e39fb97f4444c2946944482df47b39c5bbc195c54c6560b0647635f553ab23d
+    image: quay.io/redhat-appstudio/build-trusted-artifacts:latest@sha256:601e0999ae062e5ee666538d651af7893a75e415b7952c85caa8a4452501029a
```

Comment:
Is this image not available from `quay.io/konflux-ci`?

Reply:
Nope, I tried both
`quay.io/konflux-ci/build-trusted-artifacts:latest` and
`quay.io/konflux-ci/tekton-catalog/build-trusted-artifacts:latest`